tcms_api/xmlrpc.py

```diff
@@ -123,6 +132,12 @@ def __init__(self, username, password, url):
             else:
                 raise RuntimeError("Unrecognized URL scheme")
 
+        if allow_unverified_ssl:
+            ssl._create_default_https_context = ssl._create_unverified_context
+
+        if len(extra_headers) > 0:
+            self.transport._headers += extra_headers
```

Comment:
This possibly overrides an existing value set for this attribute.

You can pass headers in `Transport.__init__()` so better add this in lines 128 & 130.